apps/ledger-live-mobile/e2e/specs/speculos/addAccount/AddAccountALGO.spec.ts

```diff
@@ -0,0 +1,4 @@
+import { runAddAccountTest } from "./AddAccount";
+import { Currency } from "@ledgerhq/live-common/e2e/enum/Currency";
+
+runAddAccountTest(Currency.ALGO, "B2CQA-2497, B2CQA-2653, B2CQA-2681");
```

Comment:
when your PR is merged can you please update all affected tickets from regression scope to have "Automated on:" "LLM (Detox)"? or you maybe can do it even now

Reply:
Sure